services/QuillLMS/spec/factories/schools.rb

```diff
@@ -73,6 +73,14 @@
     lower_grade { 5 }
     upper_grade { 8 }
 
+    trait :premium do
+      sequence(:nces_id) { |n| "1234567890#{n}" }
+      after(:create) do |school|
```

Comment:
I think a one-liner works here since you don't need a temp variable for subscription.
`after(create) { create(:school_subscription, school:) }`